src/tools/pontoon/users.md

```diff
@@ -29,6 +29,21 @@ In this section it’s possible to update the display name and other personal in
 * `Contact email address`: if provided, this will be displayed in the Profile page instead of the login email address.
 * `Bio`: a short text that will be displayed in the Profile page.
 
+### Appearance
+
+There are three theme options for you to choose from:
```

Comment:
Similar changes to the one I proposed. Maybe add a short intro, e.g. `The user can chose between a light and dark combination of colors for Pontoon. The three options are:`